code/addons/a11y/package.json

```diff
@@ -60,8 +60,8 @@
     "!src/**/*"
   ],
   "scripts": {
-    "check": "../../../scripts/prepare/check.ts",
-    "prep": "../../../scripts/prepare/bundle.ts"
+    "check": "node --loader esbuild-register/loader -r esbuild-register ../../../scripts/prepare/check.ts",
+    "prep": "node --loader esbuild-register/loader -r esbuild-register ../../../scripts/prepare/bundle.ts"
```

Comment:
Before, `check.ts` and `bundle.ts` used shebangs to run ts-node. I decided against using shebang since they are not supported on Windows.

Reply:
Our monorepo has never been able to run in pure Windows, only WSL. Does this fix that, or do we need to do more work to make it possible to run the monorepo in Windows?

If there's more work needed then this doesn't seem like it will have any effect anyway. I'm not against the idea, but I'm not convinced trying to support Windows in the monorepo is worthwhile at all. Nobody in the core team is using Windows regularly, so there's a big risk that we'll see a regression later breaking Windows support.

Reply:
In April/May, I worked with Ian to make the mono repo scripts work in a Windows environment. But we have never finalized this work. I didn't know, that we had a consensus not supporting Windows for running the scripts. If that's the case, I will use shebang instead

Reply:
It's also possible that contributors using windows try to contribute, fail, and move on rather than raising a stink.  But I agree, it seems like "use WSL" is satisfactory for most of them.  

Another option to run these could be [tsx](https://github.com/esbuild-kit/tsx), though it's unclear to me what benefits it has over esbuild-register other than perhaps a simpler syntax. I asked in their discussion board just in case there's more https://github.com/esbuild-kit/tsx/discussions/370.

Reply:
Yeah I'm also not quite sure what the benefits to `tsx` are, I'm fine with trying it out to see if it's slower or faster. [We do use `esbuild-register` internally too](https://github.com/storybookjs/storybook/blob/next/code/lib/core-common/src/utils/interpret-require.ts#L11-L26), but I don't think those two concepts need to be related.

Reply:
We are keeping esbuild-register for now until we encounter issues, which are resolvable by tsx or alternatives.